File: 프로그래머스/unrated/181837. 커피 심부름/커피 심부름.py
def solution(order):
    total_price = 0
    price_americano = 4500
    price_cafelatte = 5000

    for item in order:
        if item in ["iceamericano", "americanoice", "hotamericano", "americanohot", "americano", "anything"]:
            total_price += price_americano
        elif item in ["icecafelatte", "cafelatteice", "cafelatte"]:
            total_price += price_cafelatte
        elif item in ["hotcafelatte", "cafelattehot"]:
            total_price += price_cafelatte

    return total_price

File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.java
import java.util.*;
class Solution {
    public int solution(String[] order) {
        int answer = 0;
       List<String> ame = Arrays.asList("hotamericano", "americanohot", "iceamericano", "americanoice", "americano", "anything");
        List<String> latte = Arrays.asList("icecafelatte", "cafelatteice", "hotcafelatte", "cafelattehot");
        
        for(String item:order){
            if(ame.contains(item)){
                answer+=4500;
            }else{
                answer+=5000;
            }
        }
        return answer;
    }
}